### PR TITLE
[FIX] website_sale_loyalty: ewallet application fix

### PR DESCRIPTION
**Pull request by anko-odoo**
Due to https://github.com/odoo/odoo/commit/994ee3f514d55c5c01975d3c0d77663eb173e775 commit, ewallet could no
longer be applied.

### CODE REVIEW

File: addons/website_sale_loyalty/views/website_sale_templates.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
                                                 <div class="flex-grow-1">
                                                     <t t-set="program" t-value="reward.program_id"/>
                                                     <t t-set="points" t-value="coupon._format_points(website_sale_order._get_real_points_for_coupon(coupon))"/>
-                                                    <t t-if="program.program_type not in ['ewallet', 'promo_code'] and (program.trigger == 'with_code' or (program.trigger == 'auto' and program.applies_on == 'future'))">
+                                                    <t t-if="program.program_type not in ['ewallet', 'promo_code', 'loyalty'] and (program.trigger == 'with_code' or (program.trigger == 'auto' and program.applies_on == 'future'))">
                                                         <t t-if="program.program_type == 'gift_card'">
```
Comment (Feyensv) [inline addons/website_sale_loyalty/views/website_sale_templates.xml:46]: > If Loyalty program was set to be applied on future orders,
details of reward would not be shown to user.

What I understand from the diff here is that you stop showing details if the reward is of type loyalty :thinking:

Comment (anko-odoo) [reply addons/website_sale_loyalty/views/website_sale_templates.xml:46]: Okay, I think I remember why I added loyalty to the not in, basically, loyalty by default is current&future, and it has it's own code for showing the coupon, the problem is if you set loyalty to future, suddenly it meets all the requirements for the if, but inside the if there is no other if to show the description of loyalty.
Normally if loyalty is set to current or current and future it goes to other if and shows everything properly.

Comment (Feyensv) [reply addons/website_sale_loyalty/views/website_sale_templates.xml:46]: Okay, I was probably confused by the display on github and wrongly thought that a section was in another when it wasn't.  Seems clear to me now, thanks for the explanation :+1:

**Comment by Feyensv** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**